05-cond.md

```diff
@@ -65,18 +65,17 @@ which is short for "else if".
 This makes it simple to write a function that returns the sign of a number:
 
 ~~~ {.python}
-def sign(num):
-    if num > 0:
-        return 1
-    elif num == 0:
-        return 0
-    else:
-        return -1
-
-print 'sign of -3:', sign(-3)
+num = -3
+
+if num > 0:
+    print num,"is +ve"
```

Comment:
Not sure if this is just me, but I was always confused in physics class when someone wrote "+ve", and would prefer the word "positive" to this. But, I'm willing to concede this if most people prefer "+ve"!